src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private Element[] elements;
+    private int size = 0;
+
+    {
+        elements = new Element[10];
+    }
```

Comment:
```suggestion
    private Element[] elements;
    private int size;

    public StorageImpl() {
        elements = new Element[10];
    }
```
class fields always have default values

you should use constructors instead of initialization blocks